include/openssl/rsaerr.h

```diff
@@ -155,6 +155,7 @@ int ERR_load_RSA_strings(void);
 # define RSA_R_MP_EXPONENT_NOT_CONGRUENT_TO_D             169
 # define RSA_R_MP_R_NOT_PRIME                             170
 # define RSA_R_NO_PUBLIC_EXPONENT                         140
+# define RSA_R_NULL_ARGUMENT                              180
```

Comment:
There is already an `ERR_R_PASSED_NULL_PARAMETER`, why do we need to have the same but specific for RSA?

Hint: `ERR_R_` reason codes can be used globally

Reply:
Fixed